app/assets/stylesheets/content/_work_packages_table_edit.sass

```diff
@@ -82,6 +82,12 @@
     .work-package--placeholder
       padding: 0 10px 0 0px
 
+// Mark focused, non-editable read-values
+.inplace-edit--read-value.-read-only
+  &:focus, &:hover
+    color: $inplace-edit--color--disabled
+    background: $inplace-edit--bg-color--disabled
```

Comment:
This leads to a contrast ratio of 2.46:1 which fails WCAG AA.

http://webaim.org/resources/contrastchecker/